### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by segheim**

### CODE REVIEW

File: util/app/src/main/java/demos/StringUtil.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package demos;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class StringUtil {
+    public boolean isPositiveNumber(String str) {
```
Comment (senator2202) [inline util/app/src/main/java/demos/StringUtil.java:6]: - Simplify this method to one line
- Also make it static, as what we usually do with util methods

File: multi-project/core/src/main/java/com/epam/lab/core/Utils.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package com.epam.lab.core;
+
+
+import demos.StringUtil;
+
+public class Utils {
+
+    public boolean isAllPositiveNumbers(String... str) {
```
Comment (senator2202) [inline multi-project/core/src/main/java/com/epam/lab/core/Utils.java:8]: Make this method static.

File: multi-project/core/src/main/java/com/epam/lab/core/Utils.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        StringUtil stringUtil = new StringUtil();
+        for (int i = 0; i < str.length; i++) {
+            if (!stringUtil.isPositiveNumber(str[i])) {
+                System.out.println(str[i]);
```
Comment (senator2202) [inline multi-project/core/src/main/java/com/epam/lab/core/Utils.java:12]: Remove this printing. No need to print something in utils method. Use logging if your really need it.

**Review by senator2202 — APPROVED** [review]

**Review by senator2202 — CHANGES_REQUESTED** [review]
Please, fix all above

File: multi-project/core/src/main/java/com/epam/lab/core/Utils.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package com.epam.lab.core;
+
+
+import demos.StringUtil;
+
+public class Utils {
+
+    public static boolean isAllPositiveNumbers(String... str) {
+        for (int i = 0; i < str.length; i++) {
```
Comment (senator2202) [inline multi-project/core/src/main/java/com/epam/lab/core/Utils.java:9]: What will happen if (str == null)?

**Review by Lavazz — APPROVED** [review]

**Review by senator2202 — APPROVED** [review]

**Pull request merged**